### ISSUE TITLE
JVM_DD_OPTS doesn't work with FTB?

### ISSUE THREAD

**Issue by altano** [itzg/dockerfiles#281]
Am I reading [this code](https://github.com/itzg/dockerfiles/blob/master/minecraft-server/start-minecraftFinalSetup#L77-L112) correct and if you're using FTB the JVM_DD_OPTS env var will be ignored?

**Comment by itzg** [comment]
Yes, you are. Hmm, since it has to inject parameters via that “settings-local” file, I could shift the DD parameters conversion earlier and inject those also.

**Comment by altano** [comment]
That would be lovely. If my bash-fu weren't so weak I'd submit a PR :(

**Comment by itzg** [comment]
I appreciate the PR thought :).

**Comment by itzg** [comment]
I'm very glad to hear that worked since I had to test indirectly.

Mmm, cheesecake. (If you're serious, shoot me an email at itzgeoff@gmail.com)
